### ISSUE TITLE
[MAS3.3.2] [Labels and Relationship-Settings] Legends are not associated with the check boxes 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1769]
**Actual Result:**
Legends are not associated with the check boxes and narrator also not narrating the legends information while navigating to the checkboxes 
1.User settings 2. Application updates 3. Data collection 

**Expected Result:**
Legends should be associated with the check boxes and should narrate the legends information while navigating to the checkboxes
1.User settings 2. Application updates 3. Data collection 

 **Repro Steps:**
1.Open the Bot Framework V4 Emulator.
2.Navigate to Welcome tab screen and select it, screen gets opened  
3.Navigate to the open bot and select it  
4.Open bot dialog gets opened and connect to the open bot 
5.Live chat screen gets appears 
6.Navigate to the Edit ngrok settings and select it 
7.Emulator screen gets opened 
8.Navigate to the “1. User settings 2. Application updates 3. Data collection” section check boxes 
9.Observe the issue

**User Impact:**
User who rely on AT will not able to get the labels information if the labels are not associated with the checkboxes might get confused while navigating to the Emulator settings pane.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Browser: Edge
Application: Bot Framework V4 Emulator
Screen reader: Narrator 

**MAS Reference Link:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}](url)
[MAS3.3.2_Setting.zip](https://github.com/microsoft/BotFramework-Emulator/files/3529737/MAS3.3.2_Setting.zip)
![MAS3 3 2_Setting](https://user-images.githubusercontent.com/53518994/63509445-cc992480-c4f9-11e9-9190-aec2c1c5f5eb.png)

**Comment by corinagum** [comment]
Closing as dupe of #1768
